{
    "data": [
        {
            "test_name": "mantidqt_qt5.test_samplelogs_model.test_samplelogs_model",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "mantidqt_qt5.test_samplelogs_model.test_samplelogs_model",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "23.86",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nLoad-[Notice] Load started\nLoadNexusLogs-[Warning] Sample Log &quot;cryo_temp1&quot; and 1 others contain invalid values, click &quot;Show Sample Logs&quot; for details.\nLoad-[Notice] Load successful, Duration 1.29 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.15 seconds\n.Load-[Notice] Load started\nLoadNexusLogs-[Warning] Sample Log &quot;cryo_temp1&quot; and 1 others contain invalid values, click &quot;Show Sample Logs&quot; for details.\nLoad-[Notice] Load successful, Duration 1.34 seconds\nELoad-[Notice] Load started\nLoadNexusProcessed-[Warning] Could not find group class NXsource\nLoad-[Notice] Load successful, Duration 0.72 seconds\nLoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.53 seconds\nELoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.35 seconds\nELoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.01 seconds\nLoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.33 seconds\nCreateMDWorkspace-[Notice] CreateMDWorkspace started\nCreateMDWorkspace-[Notice] CreateMDWorkspace successful, Duration 0.00 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.10 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 2.50 seconds\nE\n======================================================================\nERROR: test_get_value_for_filtered (test_samplelogs_model.SampleLogsModelTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/test/test_samplelogs_model.py&quot;, line 163, in test_get_value_for_filtered\n    self.assertEqual(get_value(all_logs[31]), &quot;{} (1 entry)&quot;.format(model.get_statistics(&quot;cryo_temp1&quot;).mean))\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/model.py&quot;, line 197, in get_statistics\n    return log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatFilteredTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_get_value_for_unfiltered (test_samplelogs_model.SampleLogsModelTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/test/test_samplelogs_model.py&quot;, line 203, in test_get_value_for_unfiltered\n    self.assertEqual(get_value(all_logs_T[2]), &quot;{} (1 entry)&quot;.format(model_T.get_statistics(&quot;ChopperStatus1&quot;).mean))\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/model.py&quot;, line 197, in get_statistics\n    return log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_model (test_samplelogs_model.SampleLogsModelTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/test/test_samplelogs_model.py&quot;, line 43, in test_model\n    stats = model.get_statistics(&quot;w&quot;)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/model.py&quot;, line 197, in get_statistics\n    return log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatFilteredTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_model_with_filtered_data (test_samplelogs_model.SampleLogsModelTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/test/test_samplelogs_model.py&quot;, line 102, in test_model_with_filtered_data\n    stats = model.get_statistics(&quot;raw_uah_log&quot;, filtered=True)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqt/mantidqt/widgets/samplelogs/model.py&quot;, line 197, in get_statistics\n    return log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatFilteredTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n----------------------------------------------------------------------\nRan 8 tests in 17.031s\n\nFAILED (errors=4)\n\n          Start 1796: PSISINQTest_PoldiFitPeaks1DTest\n"
            }
        },
        {
            "test_name": "python.MuonQt5.fit_information_test.fit_information_test",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 3,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.MuonQt5.fit_information_test.fit_information_test",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "7.05",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\n.............EE\n======================================================================\nERROR: test_time_series_log_value_from_fit_with_multiple_workspaces_uses_average_of_time_average (fit_information_test.FitInformationTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/test/Muon/fit_information_test.py&quot;, line 190, in test_time_series_log_value_from_fit_with_multiple_workspaces_uses_average_of_time_average\n    self.assertAlmostEqual(all_average, fit.log_value(&quot;ts_1&quot;), places=6)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 328, in log_value\n    return self.log_value_and_error(log_name)[0]\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 315, in log_value_and_error\n    stats = value_from_workspace(output_workspace)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 305, in value_from_workspace\n    stats = prop.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_time_series_log_value_from_fit_with_single_workspace_uses_time_average (fit_information_test.FitInformationTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/test/Muon/fit_information_test.py&quot;, line 172, in test_time_series_log_value_from_fit_with_single_workspace_uses_time_average\n    self.assertAlmostEqual(time_average, fit.log_value(&quot;ts_1&quot;), places=6)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 328, in log_value\n    return self.log_value_and_error(log_name)[0]\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 315, in log_value_and_error\n    stats = value_from_workspace(output_workspace)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 305, in value_from_workspace\n    stats = prop.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n----------------------------------------------------------------------\nRan 15 tests in 0.086s\n\nFAILED (errors=2)\n\n          Start 1802: PSISINQTest_PoldiPeakSearchTest\n"
            }
        },
        {
            "test_name": "python.MuonQt5.results_tab_model_test.results_tab_model_test",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 3,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.MuonQt5.results_tab_model_test.results_tab_model_test",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "9.29",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.10 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.10 seconds\nLoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.12 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nAddTimeSeriesLog-[Notice] AddTimeSeriesLog started\nAddTimeSeriesLog-[Notice] AddTimeSeriesLog successful, Duration 0.00 seconds\nAddTimeSeriesLog-[Notice] AddTimeSeriesLog started\nAddTimeSeriesLog-[Notice] AddTimeSeriesLog successful, Duration 0.00 seconds\nAddTimeSeriesLog-[Notice] AddTimeSeriesLog started\nAddTimeSeriesLog-[Notice] AddTimeSeriesLog successful, Duration 0.00 seconds\nLoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.09 seconds\nE.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.09 seconds\n...Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.09 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.11 seconds\nLoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.11 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.09 seconds\nLoad-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.11 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 0.16 seconds\n......\n======================================================================\nERROR: test_create_results_table_with_logs_selected (results_tab_model_test.ResultsTabModelTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/test/Muon/results_tab_widget/results_tab_model_test.py&quot;, line 269, in test_create_results_table_with_logs_selected\n    table = model.create_results_table(self.log_names, selected_results)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/results_tab_widget/results_tab_model.py&quot;, line 152, in create_results_table\n    row_dict = self._add_logs_to_table(row_dict, fit, log_selection)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/results_tab_widget/results_tab_model.py&quot;, line 177, in _add_logs_to_table\n    values = fit.log_value_and_error(log_name)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 315, in log_value_and_error\n    stats = value_from_workspace(output_workspace)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/fitting_context.py&quot;, line 305, in value_from_workspace\n    stats = prop.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n----------------------------------------------------------------------\nRan 17 tests in 2.131s\n\nFAILED (errors=1)\n\n          Start 1087: GeometryTest_BoundingBoxTest\n"
            }
        },
        {
            "test_name": "python.algorithms.GetEiT0atSNSTest.GetEiT0atSNSTest",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.algorithms.GetEiT0atSNSTest.GetEiT0atSNSTest",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "9.16",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nLoad-[Notice] Load started\nLoadNexusProcessed-[Warning] Error loading Instrument section of nxs file\nLoadNexusProcessed-[Warning] NXopengroup(sample, NXsample) failed\nLoadNexusProcessed-[Warning] Try running LoadInstrument Algorithm on the Workspace to update the geometry\nWorkspaceHistory-[Warning] Error opening the algorithm history field &#x27;process&#x27;. Workspace will have no history.\nLoad-[Notice] Load successful, Duration 0.06 seconds\nLoadInstrument-[Notice] LoadInstrument started\nLoadInstrument-[Notice] LoadInstrument successful, Duration 1.36 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nGetEiT0atSNS-[Notice] GetEiT0atSNS started\nGetEiT0atSNS-[Error] Error in execution of algorithm GetEiT0atSNS:\nGetEiT0atSNS-[Error] No energy guess was given or could be found in sample logs\nGetEiT0atSNS-[Error]   at line 52 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/GetEiT0atSNS.py&#x27;\nE\n======================================================================\nERROR: testGETS (GetEiT0atSNSTest.GetEiT0atSNSTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1070, in __call__\n    algm.execute()\nRuntimeError: No energy guess was given or could be found in sample logs\n  at line 52 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/GetEiT0atSNS.py&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/plugins/algorithms/GetEiT0atSNSTest.py&quot;, line 23, in testGETS\n    res = GetEiT0atSNS(w)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1077, in __call__\n    raise RuntimeError(msg) from e\nRuntimeError: GetEiT0atSNS-v1: No energy guess was given or could be found in sample logs\n  at line 52 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/GetEiT0atSNS.py&#x27;\n\n----------------------------------------------------------------------\nRan 1 test in 1.449s\n\nFAILED (errors=1)\n\n          Start 1527: python.algorithms.HB3AIntegratePeaksTest.HB3AIntegratePeaksTest\n"
            }
        },
        {
            "test_name": "python.algorithms.HB3AIntegrateDetectorPeaksTest.HB3AIntegrateDetectorPeaksTest",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.algorithms.HB3AIntegrateDetectorPeaksTest.HB3AIntegrateDetectorPeaksTest",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "18.35",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nHB3AAdjustSampleNorm-[Notice] HB3AAdjustSampleNorm started\nLoadMD-[Notice] LoadMD started (child)\nLoadMD-[Notice] LoadMD successful, Duration 6.52 seconds\nSetGoniometer-[Notice] SetGoniometer started (child)\nSetGoniometer-[Notice] SetGoniometer successful, Duration 0.00 seconds\nRenameWorkspace-[Notice] RenameWorkspace started (child)\nRenameWorkspace-[Notice] RenameWorkspace successful, Duration 0.00 seconds\nHB3AAdjustSampleNorm-[Notice] HB3AAdjustSampleNorm successful, Duration 6.53 seconds\nHB3AIntegrateDetectorPeaks-[Notice] HB3AIntegrateDetectorPeaks started\nRun-[Warning] Goniometer angle changed in omega log from 12.4087 to 13.609.  Used time averaged value = 12.958983333333331.\nRun-[Warning] To set to last angle, replace omega with 13.609: SetGoniometer(Workspace=&#x27;workspace&#x27;,Axis0=omega,0,1,0,1&#x27;,Axis1=&#x27;chi,0,0,1,1&#x27;,Axis2=&#x27;phi,0,1,0,1&#x27;)\nHB3AIntegrateDetectorPeaks-[Error] Error in execution of algorithm HB3AIntegrateDetectorPeaks:\nHB3AIntegrateDetectorPeaks-[Error] Python argument types in\nHB3AIntegrateDetectorPeaks-[Error]     FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\nHB3AIntegrateDetectorPeaks-[Error] did not match C++ signature:\nHB3AIntegrateDetectorPeaks-[Error]     getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\nHB3AIntegrateDetectorPeaks-[Error]   at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\nEHB3AIntegrateDetectorPeaks-[Notice] HB3AIntegrateDetectorPeaks started\nHB3AIntegrateDetectorPeaks-[Error] Error in execution of algorithm HB3AIntegrateDetectorPeaks:\nHB3AIntegrateDetectorPeaks-[Error] Python argument types in\nHB3AIntegrateDetectorPeaks-[Error]     FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\nHB3AIntegrateDetectorPeaks-[Error] did not match C++ signature:\nHB3AIntegrateDetectorPeaks-[Error]     getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\nHB3AIntegrateDetectorPeaks-[Error]   at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\nEHB3AIntegrateDetectorPeaks-[Notice] HB3AIntegrateDetectorPeaks started\nHB3AIntegrateDetectorPeaks-[Warning] Failed to fit workspace data: Output Status=success, ChiSq=22.767351197712483\nHB3AIntegrateDetectorPeaks-[Notice] HB3AIntegrateDetectorPeaks successful, Duration 1.10 seconds\nHB3AIntegrateDetectorPeaks-[Notice] HB3AIntegrateDetectorPeaks started\nHB3AIntegrateDetectorPeaks-[Error] Error in execution of algorithm HB3AIntegrateDetectorPeaks:\nHB3AIntegrateDetectorPeaks-[Error] Python argument types in\nHB3AIntegrateDetectorPeaks-[Error]     FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\nHB3AIntegrateDetectorPeaks-[Error] did not match C++ signature:\nHB3AIntegrateDetectorPeaks-[Error]     getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\nHB3AIntegrateDetectorPeaks-[Error]   at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\nEDeleteWorkspace-[Notice] DeleteWorkspace started\nDeleteWorkspace-[Notice] DeleteWorkspace successful, Duration 0.00 seconds\n\n======================================================================\nERROR: testIntegratePeaksCounts (HB3AIntegrateDetectorPeaksTest.HB3ADetectorPeaksTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1070, in __call__\n    algm.execute()\nRuntimeError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n  at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/plugins/algorithms/HB3AIntegrateDetectorPeaksTest.py&quot;, line 116, in testIntegratePeaksCounts\n    peaks = HB3AIntegrateDetectorPeaks(data, Method=&quot;Counts&quot;, ApplyLorentz=False, OptimizeQVector=False)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1077, in __call__\n    raise RuntimeError(msg) from e\nRuntimeError: HB3AIntegrateDetectorPeaks-v1: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n  at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\n\n======================================================================\nERROR: testIntegratePeaksCountsWithFitting (HB3AIntegrateDetectorPeaksTest.HB3ADetectorPeaksTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1070, in __call__\n    algm.execute()\nRuntimeError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n  at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/plugins/algorithms/HB3AIntegrateDetectorPeaksTest.py&quot;, line 162, in testIntegratePeaksCountsWithFitting\n    peaks = HB3AIntegrateDetectorPeaks(data, Method=&quot;CountsWithFitting&quot;, ApplyLorentz=False, OptimizeQVector=False, ChiSqMax=100)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1077, in __call__\n    raise RuntimeError(msg) from e\nRuntimeError: HB3AIntegrateDetectorPeaks-v1: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n  at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\n\n======================================================================\nERROR: testIntegratePeaksFitted (HB3AIntegrateDetectorPeaksTest.HB3ADetectorPeaksTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1070, in __call__\n    algm.execute()\nRuntimeError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n  at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/plugins/algorithms/HB3AIntegrateDetectorPeaksTest.py&quot;, line 28, in testIntegratePeaksFitted\n    peaks = HB3AIntegrateDetectorPeaks(data, ChiSqMax=100, SignalNoiseMin=100, ApplyLorentz=False, OptimizeQVector=False)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1077, in __call__\n    raise RuntimeError(msg) from e\nRuntimeError: HB3AIntegrateDetectorPeaks-v1: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n  at line 231 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HB3AIntegrateDetectorPeaks.py&#x27;\n\n----------------------------------------------------------------------\nRan 3 tests in 10.747s\n\nFAILED (errors=3)\n\n          Start 1522: python.algorithms.GroupBySampleChangerPositionTest.GroupBySampleChangerPositionTest\n"
            }
        },
        {
            "test_name": "python.algorithms.HFIRSANS2WavelengthTest.HFIRSANS2WavelengthTest",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.algorithms.HFIRSANS2WavelengthTest.HFIRSANS2WavelengthTest",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "6.59",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nHFIRSANS2Wavelength-[Notice] HFIRSANS2Wavelength started\nHFIRSANS2Wavelength-[Error] Error in execution of algorithm HFIRSANS2Wavelength:\nHFIRSANS2Wavelength-[Error] Could not read wavelength and wavelength_spread logs from the workspace\nHFIRSANS2Wavelength-[Error]   at line 37 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HFIRSANS2Wavelength.py&#x27;\nE\n======================================================================\nERROR: testTOF (HFIRSANS2WavelengthTest.HFIRSANS2WavelengthTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1070, in __call__\n    algm.execute()\nRuntimeError: Could not read wavelength and wavelength_spread logs from the workspace\n  at line 37 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HFIRSANS2Wavelength.py&#x27;\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/plugins/algorithms/HFIRSANS2WavelengthTest.py&quot;, line 22, in testTOF\n    out = HFIRSANS2Wavelength(InputWorkspace=&quot;ws&quot;)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/mantid/simpleapi.py&quot;, line 1077, in __call__\n    raise RuntimeError(msg) from e\nRuntimeError: HFIRSANS2Wavelength-v1: Could not read wavelength and wavelength_spread logs from the workspace\n  at line 37 in &#x27;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/plugins/algorithms/HFIRSANS2Wavelength.py&#x27;\n\n----------------------------------------------------------------------\nRan 1 test in 0.007s\n\nFAILED (errors=1)\n\n          Start  963: DataHandlingTest_RenameLogTest\n"
            }
        },
        {
            "test_name": "python.kernel.TimeSeriesPropertyTest.TimeSeriesPropertyTest",
            "latest_failed_build": "1351",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.kernel.TimeSeriesPropertyTest.TimeSeriesPropertyTest",
                "build": "1351",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "6.10",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nEEEE\n======================================================================\nERROR: test_time_series_bool_can_be_extracted (TimeSeriesPropertyTest.TimeSeriesPropertyTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 90, in test_time_series_bool_can_be_extracted\n    self._check_has_time_series_attributes(log_series)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 105, in _check_has_time_series_attributes\n    stats = log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    BoolTimeSeriesProperty.getStatistics(BoolTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;bool&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_time_series_double_can_be_extracted (TimeSeriesPropertyTest.TimeSeriesPropertyTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 66, in test_time_series_double_can_be_extracted\n    self._check_has_time_series_attributes(log_series)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 105, in _check_has_time_series_attributes\n    stats = log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    FloatTimeSeriesProperty.getStatistics(FloatTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;double&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_time_series_int_can_be_extracted (TimeSeriesPropertyTest.TimeSeriesPropertyTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 74, in test_time_series_int_can_be_extracted\n    self._check_has_time_series_attributes(log_series)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 105, in _check_has_time_series_attributes\n    stats = log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    Int64TimeSeriesProperty.getStatistics(Int64TimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;long long&gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n======================================================================\nERROR: test_time_series_string_can_be_extracted (TimeSeriesPropertyTest.TimeSeriesPropertyTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 82, in test_time_series_string_can_be_extracted\n    self._check_has_time_series_attributes(log_series, list)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/kernel/TimeSeriesPropertyTest.py&quot;, line 105, in _check_has_time_series_attributes\n    stats = log.getStatistics()\nBoost.Python.ArgumentError: Python argument types in\n    StringTimeSeriesProperty.getStatistics(StringTimeSeriesProperty)\ndid not match C++ signature:\n    getStatistics(Mantid::Kernel::TimeSeriesProperty&lt;std::__1::basic_string&lt;char, std::__1::char_traits&lt;char&gt;, std::__1::allocator&lt;char&gt; &gt; &gt; {lvalue}, Mantid::Kernel::TimeROI const* self)\n\n----------------------------------------------------------------------\nRan 4 tests in 4.986s\n\nFAILED (errors=4)\n\n          Start  949: DataHandlingTest_LoadTOFRawNexusTest\n"
            }
        },
        {
            "test_name": "MDAlgorithmsTest_ConvertToMDMinMaxLocalTest",
            "latest_failed_build": "1350",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "MDAlgorithmsTest_ConvertToMDMinMaxLocalTest",
                "build": "1350",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "0.82",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev187\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nDllOpen-[Error] Could not open library /Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib: dlopen(/Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib, 10): Symbol not found: _PyBaseObject_Type\n  Referenced from: /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n  Expected in: flat namespace\n in /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\nRunning 7 tests.ConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal started\nConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal successful, Duration 0.01 seconds\n.ConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal started\nConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal successful, Duration 0.00 seconds\n.ConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal started\nConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal successful, Duration 0.00 seconds\n.ConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal started\nConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal successful, Duration 0.00 seconds\n.ConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal started\nConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal successful, Duration 0.00 seconds\n.ConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal started\nConvertToMDMinMaxLocal-[Notice] ConvertToMDMinMaxLocal successful, Duration 0.00 seconds\n\nIn ConvertToMDMinMaxLocalTest::test_elastic1DandExtra:\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/MDAlgorithms/test/ConvertToMDMinMaxLocalTest.h:187: Error: Expected (alg.getPropertyValue(&quot;MinValues&quot;) == &quot;0.12187,7.69667&quot;), found (&quot;0.12187,8.41&quot; != 0.12187,7.69667)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/MDAlgorithms/test/ConvertToMDMinMaxLocalTest.h:188: Error: Expected (alg.getPropertyValue(&quot;MaxValues&quot;) == &quot;0.126745,7.69667&quot;), found (&quot;0.126745,8.41&quot; != 0.126745,7.69667)\nFailed 1 of 7 tests\nSuccess rate: 85%\n\n          Start  636: CrystalTest_OptimizeLatticeForCellTypeTest\n"
            }
        },
        {
            "test_name": "python.algorithms.ExportSampleLogsToHDF5Test.ExportSampleLogsToHDF5Test",
            "latest_failed_build": "1350",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.algorithms.ExportSampleLogsToHDF5Test.ExportSampleLogsToHDF5Test",
                "build": "1350",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "6.50",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev187\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 started\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 successful, Duration 0.01 seconds\n..CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 started\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 successful, Duration 0.00 seconds\n.CreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 started\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 successful, Duration 0.00 seconds\nFCreateWorkspace-[Notice] CreateWorkspace started\nCreateWorkspace-[Notice] CreateWorkspace successful, Duration 0.00 seconds\nAddSampleLog-[Notice] AddSampleLog started\nAddSampleLog-[Notice] AddSampleLog successful, Duration 0.00 seconds\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 started\nExportSampleLogsToHDF5-[Notice] ExportSampleLogsToHDF5 successful, Duration 0.00 seconds\n.\n======================================================================\nFAIL: test_timeSeriesAreTimeAveraged (ExportSampleLogsToHDF5Test.ExportSampleLogsToHDF5Test)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/plugins/algorithms/ExportSampleLogsToHDF5Test.py&quot;, line 68, in test_timeSeriesAreTimeAveraged\n    self.assertEqual(logs_group[&quot;TestLog&quot;][()], 1.5)\nAssertionError: array([2.], dtype=float32) != 1.5\n\n----------------------------------------------------------------------\nRan 5 tests in 0.050s\n\nFAILED (failures=1)\n\n          Start 2208: python.MuonQt5.pairing_table_alpha_test.pairing_table_alpha_test\n"
            }
        },
        {
            "test_name": "python.api.RunTest.RunTest",
            "latest_failed_build": "1350",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.api.RunTest.RunTest",
                "build": "1350",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "1.01",
                "stacktrace": "..............F\n======================================================================\nFAIL: test_timestd (RunTest.RunTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/PythonInterface/test/python/mantid/api/RunTest.py&quot;, line 134, in test_timestd\n    self.assertEqual(run.getTimeAveragedStd(&quot;TEMP1&quot;), expected)\nAssertionError: 5.744562646538029 != 5.163977794943222\n\n----------------------------------------------------------------------\nRan 15 tests in 0.007s\n\nFAILED (failures=1)\n\n          Start 1632: python.WorkflowAlgorithms.FitGaussianPeaksTest.FitGaussianPeaksTest\n"
            }
        },
        {
            "test_name": "APITest_LogManagerTest",
            "latest_failed_build": "1347",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "APITest_LogManagerTest",
                "build": "1347",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "1.95",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev184\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nDllOpen-[Error] Could not open library /Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib: dlopen(/Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib, 10): Symbol not found: _PyBaseObject_Type\n  Referenced from: /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n  Expected in: flat namespace\n in /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\nRunning 47 tests.................................TimeSeriesProperty-[Notice] Log series has 1 entries removed due to duplicated time. \n\nIn LogManagerTest::test_GetPropertyAsSingleValue_Returns_Correct_SingleValue_For_Each_StatisticType:\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/API/test/LogManagerTest.h:392: Error: Expected (runInfo.getPropertyAsSingleValue(name, Math::TimeAveragedMean) == TIME_AVG_MEAN) up to 1e-12 (0.0000000000), found (18.2380952348 != 15.3571428591)\n.............\nFailed 1 of 47 tests\nSuccess rate: 97%\n\n          Start 1030: DataObjectsTest_EventWorkspaceTest\n"
            }
        },
        {
            "test_name": "DataHandlingTest_LoadILLDiffractionTest",
            "latest_failed_build": "1346",
            "agent_name": "Mac",
            "last_failed_outcome": "Flake",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 4,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Flake",
                    "Flake",
                    "Flake",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Flake",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "DataHandlingTest_LoadILLDiffractionTest",
                "build": "1346",
                "agent_name": "Mac",
                "trial": 1,
                "outcome": "Failed",
                "test_time": "12.21",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.5.20230127.1705.dev180\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nDllOpen-[Error] Could not open library /Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib: dlopen(/Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib, 10): Symbol not found: _PyBaseObject_Type\n  Referenced from: /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n  Expected in: flat namespace\n in /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\nRunning 11 tests.DllOpen-[Error] Could not open library /Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib: dlopen(/Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib, 10): Symbol not found: _PyBaseObject_Type\n  Referenced from: /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n  Expected in: flat namespace\n in /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n...LoadILLDiffraction-[Notice] LoadILLDiffraction started\nLoadILLDiffraction-[Notice] LoadILLDiffraction successful, Duration 1.19 seconds\n.Load-[Notice] Load started\nLoad-[Notice] Load successful, Duration 1.02 seconds\n.InstrumentDefinitionParser-[Notice] Geometry cache is not available\nInstrumentDefinitionParser-[Notice] Creating cache in /Users/builder/.mantid/instrument/geometryCache/D2Bfed5c17191ce18f4c6a528b1c0feee2d87e63c68.vtp\n\nIn LoadILLDiffractionTest::test_D2B_alignment:\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/DataHandling/test/LoadILLDiffractionTest.h:358: Error: Assertion failed: !outputWS-&gt;detectorInfo().isMonitor(16385 * 25)\n.InstrumentDefinitionParser-[Notice] Geometry cache is not available\nInstrumentDefinitionParser-[Notice] Creating cache in /Users/builder/.mantid/instrument/geometryCache/D2B3b7f429c025fbdc995eaf67c0329ff427a57bfbe.vtp\n.LoadILLDiffraction-[Notice] A 2theta offset angle is necessary for D1B data.\nInstrumentDefinitionParser-[Notice] Geometry cache is not available\nInstrumentDefinitionParser-[Notice] Creating cache in /Users/builder/.mantid/instrument/geometryCache/D1B8d3124501c775196e31d5ce933bb9a51db0c969e.vtp\n..\nFailed 1 of 11 tests\nSuccess rate: 90%\n\n          Start  887: DataHandlingTest_LoadILLDiffractionTest\n"
            }
        },
        {
            "test_name": "workbench.test_mantidfigurecanvas.test_mantidfigurecanvas",
            "latest_failed_build": "1343",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "1363",
                    "1362",
                    "1361",
                    "1360",
                    "1359",
                    "1358",
                    "1357",
                    "1356",
                    "1355",
                    "1354",
                    "1353",
                    "1352",
                    "1351",
                    "1350",
                    "1349",
                    "1348",
                    "1347",
                    "1346",
                    "1345",
                    "1344",
                    "1343",
                    "1342",
                    "1341",
                    "1340",
                    "1339",
                    "1338",
                    "1337",
                    "1336",
                    "1335",
                    "1334"
                ],
                "past_outcome_list": [
                    "None",
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed"
                ]
            },
            "last_stack_traces": {
                "test_name": "workbench.test_mantidfigurecanvas.test_mantidfigurecanvas",
                "build": "1343",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "1.40",
                "stacktrace": "F\n======================================================================\nFAIL: test_dpi_attribute_exists (test_mantidfigurecanvas.MantidFigureCanvasTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/qt/applications/workbench/workbench/plotting/test/test_mantidfigurecanvas.py&quot;, line 19, in test_dpi_attribute_exists\n    self.assertTrue(hasattr(canvas, &quot;_dpi_ratio&quot;))\nAssertionError: False is not true\n\n----------------------------------------------------------------------\nRan 1 test in 0.208s\n\nFAILED (failures=1)\n\n          Start 2503: workbench.test_recoveryfailureview.test_recoveryfailureview\n"
            }
        }
    ]
}